<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="Rosamistica" failures="1" tests="8" name="Test" time="48.215" errors="0" timestamp="2023-07-17T13:31:31 ADT">
  <testcase classname="naveenautomationlabs.AutomationFramework.Test.AppleCinema30Test" name="validateCheckOut" time="0.008"/>
  <testcase classname="naveenautomationlabs.AutomationFramework.Test.MyAccountInformationTest" name="validateSideNavigationEditAccount" time="2.392"/>
  <testcase classname="naveenautomationlabs.AutomationFramework.Test.YourStoreTest" name="validateCheckOut" time="0.0"/>
  <testcase classname="naveenautomationlabs.AutomationFramework.Test.MonitorsPageTest" name="validateCheckOut" time="0.001"/>
  <testcase classname="naveenautomationlabs.AutomationFramework.Test.MyAccountSideNavigationTest" name="validateSideNavigationMyAccount" time="1.795"/>
  <testcase classname="naveenautomationlabs.AutomationFramework.Test.MyAccountSideNavigationTest" name="validateTestCase" time="0.0">
    <skipped/>
  </testcase> <!-- validateTestCase -->
  <testcase classname="naveenautomationlabs.AutomationFramework.Test.MyAccountSideNavigationTest" name="validateTestCase" time="0.002">
    <skipped/>
  </testcase> <!-- validateTestCase -->
  <testcase classname="naveenautomationlabs.AutomationFramework.Test.MyAccountSideNavigationTest" name="validateTestCase" time="0.001">
    <failure type="java.lang.AssertionError" message="Failed test expected [Test1] but found [Test]">
      <![CDATA[java.lang.AssertionError: Failed test expected [Test1] but found [Test]
at naveenautomationlabs.AutomationFramework.Test.MyAccountSideNavigationTest.validateTestCase(MyAccountSideNavigationTest.java:38)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- validateTestCase -->
</testsuite> <!-- Test -->
